concent_api/core/tests/utils.py

```diff
@@ -637,45 +616,16 @@ def _get_serialized_subtask_results_rejected(
 
     def _get_deserialized_compute_task_def(  # pylint: disable=no-self-use
         self,
-        task_id: Optional[str] = None,
-        subtask_id: Optional[str] = None,
-        deadline: Union[str, int, float, None] = None,
-        extra_data: Optional[dict] = None,
-        short_description: str = 'path_root: /home/dariusz/Documents/tasks/resources, start_task: 6, end_task: 6...',
-        performance: float = 829.7531773625524,
-        docker_images: Optional[List[set]] = None,
-        frames: Optional[List[int]] = None
+        kwargs: Dict[str, Union[int, bytes, str]] = None,
     ) -> ComputeTaskDef:
-        compute_task_def = ComputeTaskDefFactory(
-            task_id=task_id if task_id is not None else self._get_uuid(),
-            subtask_id=subtask_id if subtask_id is not None else self._get_uuid(),
-            extra_data=extra_data,
-            short_description=short_description,
-            performance=performance,
-        )
-        if isinstance(deadline, (int, float)):
-            compute_task_def['deadline'] = deadline
-        elif isinstance(deadline, str):
-            compute_task_def['deadline'] = parse_iso_date_to_timestamp(deadline)
-        else:
-            compute_task_def['deadline'] = parse_iso_date_to_timestamp(get_timestamp_string()) + 10
-
-        if extra_data is None:
-            compute_task_def['extra_data'] = {
-                'end_task': 6,
-                'frames': frames if frames is not None else [1],
-                'outfilebasename': 'Heli-cycles(3)',
-                'output_format': 'PNG',
-                'path_root': '/home/dariusz/Documents/tasks/resources',
-                'scene_file': '/golem/resources/scene-Helicopter-27-internal.blend',
-                'script_src': '# This template is rendered by',
-                'start_task': 6,
-                'total_tasks': 8
-            }
-        if docker_images is None:
-            compute_task_def['docker_images'] = [{'image_id': None, 'repository': 'golemfactory/blender', 'tag': '1.4'}]
-
-        return compute_task_def
+        if not isinstance(kwargs, dict):
+            kwargs = {}
```

Comment:
please change to `Optional` Dict in function declaration and here check simple if `kwargs` is `None`. If it will be other type it should fail with errer